hw/ip/lc_ctrl/data/lc_ctrl.hjson

```diff
@@ -335,6 +335,24 @@
     { name: "BUS.INTEGRITY",
       desc: "End-to-end bus integrity scheme."
     }
+    { name: "ENCODING.SPARSE",
+      desc: "Sparse manufacturing state encoding."
+    }
+    { name: "MAIN.FSM.SPARSE",
+      desc: "The main state FSM is sparsely encoded."
+    }
+    { name: "KMAC.FSM.SPARSE",
+      desc: "The KMAC interface FSM is sparsely encoded."
+    }
+    { name: "MAIN.FSM.ESCALATION",
+      desc: "The main state FSM moves to virtual scrap upon escalation."
+    }
+    { name: "SIGNAL.MUBI",
```

Comment:
`SIGNAL` is currently not a possible asset. We have `LOGIC` but the recommendation is to avoid it as it is too broad. Should this rather be `CTRL`? Also, I guess this is about LC control signals fed to other modules but not about signals inside LC control, correct?

Reply:
Correction. I just noted that this PR adds `SIGNAL` as asset.

Reply:
Yeah so I believe @rswarbrick came across this as well in https://github.com/lowRISC/opentitan/pull/9638. 
We currently do not have an asset name for signals between modules that are not of type `BUS`, hence I added this.
We can also call it `INTERSIG` (or something else) if you think that would be more concise.

Reply:
sorry a bit late, INTERSIG does seem to work well.
Especially since we actually have the concept of an intersig in our python scripting.

Reply:
Thanks @tjaychen, I've changed this accordingly!

CC @rswarbrick for ROM_CTRL